### DISCUSSION TITLE
Can we change a VCF track parameter to avoid the 'stats timeout' / 'force load' button?

### DISCUSSION THREAD

**Discussion by bbimber** [GMOD/jbrowse-components#4714] [General]
Hello,

JBrowse has code to estimate feature density and protect the user from loading too much data. However, we have a couple of feature-dense VCFs and I would like to figure out a way to make this threshold more tolerant, since virtually any view gives the user the 'force load' button. Is there a track-level parameter that can do this?

Thanks for any help.

**Comment by cmdcolin** [comment]
i have not tested very much with files that produce stats timeouts, I should probably try this out before guessing at an answer. Do you have any openly available files that do this? I imagine i could also simulate a slow network to get it to work or hack the code to force it, but testing on real data can help sometimes

**Reply by bbimber** [reply]
if you dont mind registering here, this is otherwise public: https://mgap.ohsu.edu/project/mGAP/begin.view?pageId=overview, and then once registered: https://mgap.ohsu.edu/jbrowse/mGAP/browser.view?database=9D7297DA-5030-103D-B6A7-A4C7310EFC10

again, this just happens to be a very dense VCF with a lot of samples
